backend/FwLite/FwDataMiniLcmBridge/Api/FwDataMiniLcmApi.cs

```diff
@@ -156,6 +157,11 @@ public async IAsyncEnumerable<PartOfSpeech> GetPartsOfSpeech()
         }
     }
 
+    public async Task CreatePartOfSpeech(PartOfSpeech partOfSpeech)
+    {
+        throw new NotImplementedException();
+    }
+
     public async IAsyncEnumerable<SemanticDomain> GetSemanticDomains()
     {
         foreach (var semanticDomain in SemanticDomainRepository.AllInstances().OrderBy(p => p.Name.BestAnalysisAlternative.Text))
```

Comment:
For semantic domains, I believe sorting by abbreviation, or possibly `.AbbrAndName`, is a more natural way to present the choices to the user: that way "1. Whatever" gets shown before "1.1. Whatever subdomain" and 1.1 is shown before "2. Something else".

Reply:
that's a good point, however at the moment at least we don't display the code in the front end, so then it makes more sense sorted this way. That being said, since we always return all domains, I'd just let the frontend sort it however it wants.